irida_staramr_results/amr_downloader.py

```diff
@@ -0,0 +1,54 @@
+import io
+import os
+import logging
+
+import pandas as pd
+
+
+def download_all_results(irida_api, project_id, output_file_name):
+    """
+    Main function for downloading StarAMR results to an excel file.
+    :param irida_api:
+    :param project_id:
+    :param output_file_name:
+    :return:
+    """
+
+    logging.info(f"Requesting completed amr analysis submissions for project id [{project_id}]")
+    amr_completed_analysis_submissions = irida_api.get_amr_analysis_submissions(project_id)
+
+    if len(amr_completed_analysis_submissions) < 1:
+        logging.info(f"No completed amr analysis submission type for project id [{project_id}].")
```

Comment:
This should likely be a `warning` as nothing is getting written

Reply:
Done